File: Estrutura_Condicional/Python/Ex1047.py
hora_inicio, minuto_inicio, hora_fim, minuto_fim = map(int, input().split())

inicio_em_minutos = hora_inicio * 60 + minuto_inicio
fim_em_minutos = hora_fim * 60 + minuto_fim

duracao_em_minutos = fim_em_minutos - inicio_em_minutos

if duracao_em_minutos <= 0:
    duracao_em_minutos += 24 * 60 

duracao_horas = duracao_em_minutos // 60
duracao_minutos = duracao_em_minutos % 60

print("O JOGO DUROU", duracao_horas, "HORA(S) E", duracao_minutos, "MINUTO(S)")

File: Estrutura_Condicional/JavaScript/Ex1047.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};

var valores=prompt("valores")
    var [hInicio,mInicio,hFim,mFim]=valores.split(" ").map(Number)
    
    minutosTotais=mFim - mInicio
    horasTotais = (hFim - hInicio) * 60
    tempoTotal = horasTotais + minutosTotais
    
    if(tempoTotal <= 0){
        tempoTotal += 24*60
    }
    
    h = parseInt(tempoTotal/60)
    m = tempoTotal%60
    
    console.log("O JOGO DUROU "+h+" HORA(S) E "+m+" MINUTO(S)")